app/models/genre_term.rb

```diff
@@ -0,0 +1,93 @@
+# Copyright 2017, University of Alberta Libraries.
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+#   under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+#   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+#   specific language governing permissions and limitations under the License.
+# ---  END LICENSE_HEADER BLOCK  ---
+
+class GenreTerm
+  require 'avalon/levenshtein'
```

Comment:
Not wild about this. Add lib to the autoload and eager_load paths, or move the module under /app, in my opinion. (But I think it's worth considering whether copy-pasting this code into our codebase is the right approach, see my other comment.)